pkg/operator/controllers/guardrails/staticresources/gk_pod_security_policy.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  # annotations:
+  #   seccomp.security.alpha.kubernetes.io/allowedProfileNames: '*'
```

Comment:
this is required by gatekeeper to be installed on openshift, [the link](https://open-policy-agent.github.io/gatekeeper/website/docs/vendor-specific/#running-on-openshift-4x) is for the latest gatekeeper, I dont have it for 3.9 which is the version used in the PR.
as to why it is commented instead of removed, this is for the convenience of upgrading, this section will be easily recognised as intended instead of omitted.